torch/csrc/jit/passes/onnx/cast_all_constant_to_floating.cpp

```diff
@@ -1,3 +1,4 @@
+#include <torch/csrc/Exceptions.h>
 #include <torch/csrc/jit/passes/onnx/cast_all_constant_to_floating.h>
 #include <torch/csrc/jit/passes/onnx/helper.h>
 
```

Comment:
According to the pybind11 docs ([Header and namespace conventions](https://pybind11.readthedocs.io/en/stable/basics.html#header-and-namespace-conventions)), the default namespace is `pybind11` rather than `py`. This may be the cause of "py is not a type".

```suggestion
namespace py = pybind11;

```